server/container_create_linux.go

```diff
@@ -866,6 +868,21 @@ func setupWorkingDirectory(rootfs, mountLabel, containerCwd string) error {
 	return nil
 }
 
+func getOCIMappings(m []*types.IDMapping) []rspec.LinuxIDMapping {
```

Comment:
nit: can the function be defined below when it's called

Reply:
sure, done now